packages/next/test/fixtures/37-bundled-server/probes.json

```diff
@@ -0,0 +1,24 @@
+{
+  "probes": [
+    {
+      "path": "/app",
+      "status": 200,
+      "mustContain": "hello world"
+    },
+    {
+      "path": "/app-route",
+      "status": 200,
+      "mustContain": "hello world"
+    },
+    {
+      "path": "/api/hello",
+      "status": 200,
+      "mustContain": "hello world"
+    },
+    {
+      "path": "/",
+      "status": 200,
+      "mustContain": "hello world"
```

Comment:
All of these tests are looking for the same text. Can we update each so they return something different that that will ensure it works as expected?